index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Rosetube</title>
+        <link rel = "stylesheet" href="style.css">
+
+    </head>
+    <body>
+        <nav class="flex-div">
+            <div class=" nav-left flex-div">
+                <img src="images/menu.png" class="menu-icon">
+                <img src="images/logo.png" class="logo">
+            </div>
+            <div class=" nav-middle flex-div">
+                <div class="search-box flex-div">
+                    <input type="text" placeholder="search">
+                    <img src="images/search.png">
+                </div>
+                <img src="images/voice-search.png" class="mic-icon">
+            </div>
+            <div class=" nav-right flex-div">
+                <img src="images/upload.png">
+                <img src="images/more.png">
+                <img src="images/notification.png">
+                <img src="images/Jack.png" class = "user-icon">
+            </div>
+            
+            
+        </nav>
+        <!-- ---------- side bar ----- -->
+        <div class = "sidebar">
+            <div class="shortcut-links">
+                <a href=" " ><img src="images/home.png"><p>Home</p></a>
+                <a href=" " ><img src="images/explore.png"><p>Explore</p></a>
+                <a href=" " ><img src="images/subscriprion.png"><p>Subscriprion</p></a>
+                <a href=" " ><img src="images/library.png"><p>Library</p></a>
+                <a href=" " ><img src="images/playlist.png"><p>Playlist</p></a>
+                <a href=" " ><img src="images/messages.png"><p>Messages</p></a>
+                <a href=" " ><img src="images/show-more.png"><p>Show more</p></a>
+                <hr>
+            </div>
+            <div class = "subcribed_list">
+                <h3>SUBCRIBED</h3>
+                <a href=" " ><img src="images/Jack.png"><p>Jason Nicho</p></a>
+                <a href=" " ><img src="images/simon.png"><p>Simeon Kris</p></a>
+                <a href=" " ><img src="images/tom.png"><p>Tom Johnson</p></a>
+                <a href=" " ><img src="images/megan.png"><p>Megan Jack</p></a>
+                <a href=" " ><img src="images/cameron.png"><p>Cameron Olivi</p></a>
+                
+            </div>
```

Comment:
Nice work! but I will suggest you put all the a tags in ul and li. This will compact your codes and make it easier to read and debug.

Reply:
Great job being polite, phrasing your comment as a suggestion, and giving a reason for the requested change.